integration-libs/punchout/components/punchout-requisition/punchout-requisition.component.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  inject,
+  OnInit,
+  ViewChild,
+} from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { RoutingService } from '@spartacus/core';
+import {
+  PUNCHOUT_ERROR_PAGE_URL,
+  PunchoutFacade,
+  PunchoutRequisition,
+} from '@spartacus/punchout/root';
+import { filter, map, Observable, switchMap, take, tap, timer } from 'rxjs';
+
+@Component({
+  selector: 'cx-punchout-requsition',
+  templateUrl: './punchout-requisition.component.html',
+  changeDetection: ChangeDetectionStrategy.Default,
```

Comment:
Is this ChangeDetection value not `OnPush` for purpose?

Reply:
Thx, was for test purpose, i forgot to revert.